rendezvous/README.md

```diff
@@ -16,7 +16,7 @@ Interest Group: [@daviddias], [@whyrusleeping], [@Stebalien], [@jacobheun], [@yu
 [@yusefnapora]: https://github.com/yusefnapora
 [@vasco-santos]: https://github.com/vasco-santos
 
-See the [lifecycle document][lifecycle-spec] for context about maturity level
+See the [lifecycle document][lifecycle-spec] for context about the maturity level
```

Comment:
Note that this is the same across all the specifications. In case you would like to see it changed, can you please propose it in a new pull request updating all the specifications?

Reply:
Yes, planning to go through some of the other specs as an opportunity to update writing style.

Reply:
Sorry potentially coming across as nit-picky here. Let's do this change in an atomic fashion, i.e. either all at once or not at all.